src/main/java/com/configcat/LocalFilePollingMode.java

```diff
@@ -0,0 +1,30 @@
+package com.configcat;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+class LocalFilePollingMode extends PollingMode {
```

Comment:
Should we implement `getPollingIdentifier()` functon here?

Reply:
`PollingMode` gives us a default implementation which is enough here I think as it's redundant with this mode.